pkg/pillar/types/global.go

```diff
@@ -222,193 +795,281 @@ var GlobalConfigDefaults = GlobalConfig{
 
 // Check which values are set and which should come from defaults
 // Zero integers means to use default
-func ApplyGlobalConfig(newgc GlobalConfig) GlobalConfig {
+
+// ApplyDefaults - Applies defaults to an oldglobal config
+// Kept for upgradeconverter
+func ApplyDefaults(newgc OldGlobalConfig) OldGlobalConfig {
```

Comment:
Does the converter use this function or is it now unused? If so it can be removed

Reply:
The converter uses the function to create a default oldGlobalConfig

Reply:
Which you don't need because the converter can create a default new config when the old config doesn't exist (or is unreadable/unparsable)